frontend/lib/api/polling_station/usePollingStationMutation.ts

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+
+import {
+  LimitedApiRequestState,
+  POLLING_STATION_CREATE_REQUEST_BODY,
+  POLLING_STATION_CREATE_REQUEST_PATH,
+  POLLING_STATION_UPDATE_REQUEST_BODY,
+  POLLING_STATION_UPDATE_REQUEST_PATH,
+  PollingStation,
+} from "@kiesraad/api";
+import { callAsync } from "@kiesraad/util";
+
+import { useCrud } from "../useCrud";
+
+export type PollingStationSubmitCreate = (electionId: number, obj: POLLING_STATION_CREATE_REQUEST_BODY) => void;
+export type PollingStationSubmitUpdate = (pollingStationId: number, obj: POLLING_STATION_UPDATE_REQUEST_BODY) => void;
+
+export type UsePollingStationMutationReturn = {
+  create: PollingStationSubmitCreate;
+  update: PollingStationSubmitUpdate;
+  requestState: LimitedApiRequestState<PollingStation>;
+};
+
+export function usePollingStationMutation(): UsePollingStationMutationReturn {
+  const { requestState, create: crudCreate, update: crudUpdate } = useCrud<PollingStation>(true);
+
+  const create: PollingStationSubmitCreate = React.useCallback(
```

Comment:
I would make these functions async, and use the void operator when using them in the component, i.e. `void create(electionId, requestObj);` in `PollingStationForm`. Then it is more obvious in the handler that this is an async operation.
The `callAsync` util can then be removed again.

Reply:
It's specifically in the form component I would prefer not to use async code since this is not the flow we are using but are using state changes. using "both" to me seems confusing.
I see "create / update" as an "action" that changes state.

I'm fine either way, just prefer to reach a consensus first.

Reply:
i've used `void useCrud`.